blueocean-dashboard/src/main/js/creation/credentials/CredentialsApi.js

```diff
@@ -0,0 +1,102 @@
+import { capabilityAugmenter, Fetch, UrlConfig } from '@jenkins-cd/blueocean-core-js';
+import TempUtils from '../TempUtils';
+
+export class CredentialsApi {
```

Comment:
Yep, I have a JIRA to do some a number of refactorings of code into different modules, see:  https://issues.jenkins-ci.org/browse/JENKINS-41139  You mind if I land this as-is now? I expect to complete that ticket next week.